src/Humanizer.Tests/PublicApiApprovalTest.approve_public_api.approved.txt

```diff
@@ -164,6 +164,12 @@ namespace Humanizer
         Feminine = 1,
         Neuter = 2,
     }
+    public class static HeadingExtensions
+    {
+        public static double FromShortHeading(this string heading) { }
+        public static string ToHeading(this double heading, bool shortResult = False, System.Globalization.CultureInfo culture = null) { }
```

Comment:
I would prefer to see an Enum here instead of a boolean since I believe it makes the intent more clear.

I can have two values and default the same.

Reply:
Hmm that makes more sense indeed. What do you mean with the two values thing?